docs/devel/networking.md

```diff
@@ -326,3 +326,51 @@ Chain POSTROUTING (policy ACCEPT)
 target     prot opt source               destination
 MASQUERADE  all  --  10.0.2.2             anywhere
 ```
+
+### Masquerade binding using IPv6 addresses
+The masquerade binding mechanism is currently the only binding mechanism which
+accepts IPv6 addresses.
+
+It operates in exactly the same way as in IPv4, and follows the same goals:
+configure one to one NAT. As in it's IPv4 counter-part, the pods are reached
+via their IPv6 pod addresses.
+
+NAT is configured in the exact same way, but using ip6tables, or the `ipv6-nat`
+nftable table. Please refer to the tables below to visualize how NAT for IPv6
+addresses is accomplished in KubeVirt.
+
+```
+Chain PREROUTING (policy ACCEPT)
+target               prot opt source               destination
+KUBEVIRT_PREINBOUND  all      anywhere             anywhere
+
+Chain INPUT (policy ACCEPT)
+target     prot opt source               destination
+
+Chain OUTPUT (policy ACCEPT)
+target     prot opt source               destination
+DNAT       tcp      anywhere             localhost            tcp dpt:http to:fd10:0:2::2
+
+Chain POSTROUTING (policy ACCEPT)
+target     prot opt source               destination
+MASQUERADE  all      fd10:0:2::2          anywhere
+KUBEVIRT_POSTINBOUND  all      anywhere   anywhere
+
+Chain KUBEVIRT_POSTINBOUND (1 references)
+target     prot opt source               destination
+SNAT       tcp      anywhere             anywhere             tcp dpt:http to:fd10:0:2::1
+
+Chain KUBEVIRT_PREINBOUND (1 references)
+target     prot opt source               destination
+DNAT       tcp      anywhere             anywhere             tcp dpt:http to:fd10:0:2::2
+```
+
+It is important to refer that when IPv6 masquerade binding is set in place,
```

Comment:
There is no IPv6 masquerade binding. Masquerade binding supports both IPv4 and IPv6. There is no way to select one or the other. I think you could link this to the note you made at the beginning of this doc - CNI is first stage that is given to use, in binding, we push whatever CNI interface to the guest. With masquerade binding, we make sure to push both IPv4 and IPv6 traffic.

Reply:
The wording got twisted after leaving my brain and before being written down.

Is it better now ? You still want the clarification ? I'd rather just leave this as simple as possible: it's works side by side w/ IPv4, and just configures the NAT table for ipv6 using the corresponding hard-coded ipv6 addresses.